src/iop/dummy1.c

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
To be removed. It is commented out in CMakeList.txt.

The same apply to amm dummy?.c